.changeset/eleven-shoes-fix.md

```diff
@@ -0,0 +1,5 @@
+---
+"@millicast/sdk": patch
+---
+
+Bring back onMetadata
```

Comment:
This should not be a changesets, as it is a rollback

Reply:
Ok, I shall remove it.